app/src/main/java/com/example/wrk/LoginActivity.java

```diff
@@ -51,6 +51,17 @@ public void onClick(View v) {
                 loginUser(username, password);
             }
         });
+        etUsername.setTranslationX(300);
```

Comment:
the 300, 1000 numbers are reused accross files, so you could put them in a const

Reply:
Good idea. Will fix this